R/render.R

```diff
@@ -802,8 +802,12 @@ resolve_df_print <- function(df_print) {
   if (!is.function(df_print)) {
     if (df_print == "kable")
       df_print <- knitr::kable
-    else if (df_print == "tibble")
+    else if (df_print == "tibble") {
+      if (!requireNamespace("tibble", quietly = TRUE))
+        stop("Printing 'tibble' without 'tibble' package available")
```

Comment:
I wonder if instead of making this a hard error we could just use the default print method (and maybe warn once per session)?